app/src/main/java/com/github/polybooks/database/FirebaseProvider.kt

```diff
@@ -0,0 +1,17 @@
+package com.github.polybooks.database
+
+import com.google.firebase.firestore.FirebaseFirestore
+
+object FirebaseProvider {
+
+    fun getFirestore() : FirebaseFirestore = FirebaseFirestore.getInstance()
+
+    fun getMockFirestore() : FirebaseFirestore {
+        val instance = FirebaseFirestore.getInstance()
+        try { //Sorry, all the clean ways to prevent calling this twice don't work and I don't know why.
+            instance.useEmulator("10.0.2.2", 8080)
```

Comment:
What does "10.0.2.2" means ? The version of mock instance ?

Reply:
10.0.2.2 est une adresse IP qui représente l'adresse de l'émulateur. 8080 c'est le port, il y a des ports différents pour différents services firebase. pour l'authentification le port est 9099, et pour la realtime database, c'est 9000.